.github/workflows/ci.yaml

```diff
@@ -0,0 +1,38 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+
+jobs:
+  build_test:
+    runs-on: ubuntu-latest
+    env:
+      TF_ACC: "1"
+      GO111MODULE: "on"
```

Comment:
nit: this isn't needed [ref](https://maelvls.dev/go111module-everywhere/#go111module-with-go-117)

```suggestion
```